src/routes/api/v1/search-index/+server.ts

```diff
@@ -14,37 +17,39 @@ const CONTENTFUL_ACTIONS = {
   DELETE: "ContentManagement.Entry.delete",
 };
 
+const getMetaTitleAndDescription = (
+  item?: SearchIndexingMetadataMapItem,
+): { metaTitle?: string | null; metaDescription?: string | null } => {
+  let metaTitle, metaDescription;
+  if (item?.__typename == "PageMetadata") {
+    metaTitle = item?.metaTitle ?? item?.title;
+    metaDescription = item?.metaDescription;
```

Comment:
we're gonna have to fix this. There reason for the `for fields in body.fields` loop [below](https://github.com/la-ldaf/ldaf-site/pull/579/files#diff-18e3587586f7ba5303d44f50cf6bffdebc3fb35c13b16bd4d4ccc000abf1f8aaR65) is because Contentful's webhook only sends fields that have changed in the request body, so we end up skipping any properties that aren't sent in the request body. Since this explicitly checks for the title and description, this could unintentionally set fields to undefined when they haven't changed.